docs/img_utils.py

```diff
@@ -68,14 +58,14 @@ def get_paths_rel_path(paths: Set[Path], rel: Path) -> Set:
     return {f"/{path.relative_to(rel)}" for path in paths}
 
 
-def get_docs_img_path(paths: Set[Path], pattern: re.Pattern) -> Set:
+def get_docs_img_path(paths: Set[Path]) -> Set:
     """Get all img syntax from given :param:`paths` using the regexp from :param:`pattern`."""
     res = set()
     for path in paths:
         content = path.read_text()
-        find = pattern.findall(content)
+        find = re.findall(r"/img[\w./-]*", content)
```

Comment:
I think we should move this regexp pattern to L102 or L64 for higher performance. And the best practices for regexp is using `re.compile` to create one compile and then use it in the feature saving memory. you could see example in function `build_pattern`

Reply:
> I think we should move this regexp pattern to L102 or L64 for higher performance. And the best practices for regexp is using `re.compile` to create one compile and then use it in the feature saving memory. you could see example in function `build_pattern`

Sure, I will change it to use `re.compile`.

Reply:
> I think we should move this regexp pattern to L102 or L64 for higher performance. And the best practices for regexp is using `re.compile` to create one compile and then use it in the feature saving memory. you could see example in function `build_pattern`

Fixed in the latest commit.